packages/bot-web-ui/src/pages/bot-builder/quick-strategy/form-wrappers/upgraded-qs-v2/trade-constants.ts

```diff
@@ -0,0 +1,123 @@
+import { localize } from '@deriv/translations';
+import { STRATEGIES } from '../../config';
+
+export enum QsSteps {
+    StrategySelect,
+    StrategyVerified,
+    StrategyCompleted,
+}
+
+export type TStrategyValue = {
+    id: number;
+    label: string;
+    name: string;
+};
+
+export type TTradeType = {
+    id: number;
+    name: string;
+    value: TStrategyValue[];
+};
+
+const TRADE_TYPE_KIT: Record<string, TTradeType> = {
```

Comment:
can we just keep the name to `TRADE_TYPE` ?

Reply:
changed